upgrade/README.md

```diff
@@ -112,17 +112,23 @@ Upgrade sequence:
 19. If you enabled new Insights feature in step 8, please follow the post installation steps listed [here](https://docs.google.com/document/d/1Um_FvVip5GtTWdezN2wANz3QsIoGrlFS07lTxNk6Sw8/edit?tab=t.0#heading=h.odfvfs38x0e3)
 
     **Note**: If Pipeline Insights was enabled before upgrade, you need to edit the query for Currently Running Pipelines panel (inside Pipeline Insights dashboard, in the Grafana UI) and change metric name from `currently_executing_pipelines_new_ratio` to `currently_executing_pipelines_new` (do not forget to save your changes to the dashboard)
+
+    **Note**: In this version we have upgraded the grafana from 10.2.2 to 12.0.2 we can check in grafana URL.
  
-20. Go to ISD UI and check that version number has changed in the top right corner (under Help menu)
+21. Go to ISD UI and check that version number has changed in the top right corner (under Help menu)
 
-21. Wait for about 5 min for autoconfiguration to take place.
+22. Wait for about 5 min for autoconfiguration to take place.
 
-22. If required: a) Connect Spinnaker again b) Configure pipeline-promotion again. To do this, in the ISD UI:
+23. If required: a) Connect Spinnaker again b) Configure pipeline-promotion again. To do this, in the ISD UI:
       - Click setup
       - Click Spinnaker tab at the top. Check if "External Accounts" and "Pipeline-promotion" columns show "yes". If any of them is "no":
       - Click "edit" on the 3 dots on the far right. Check the values already filled in, make changes if required and click "update".
       - Restart the halyard pod by clicking "Sync Accounts to Spinnaker" in the Cloud Accounts tab or simply delete the halyard pod
 
+24. In this version we have updated the opa,sidecars and rabbitmq images we can check by using this command:
```

Comment:
let's drop this info here..i tihnk it better fits in the release notes (both internal and external)